### PR TITLE
Terraform

### PR DESCRIPTION
**Pull request by osvegn**
# New feature
Create a bonus!
The server and web app are available over internet. They are deployed on azure using terraform

### CODE REVIEW

File: Web/release/client.apk [hunk 1]
```diff
@@ -0,0 +1 @@
+salut
```
Comment (31Nathan) [inline Web/release/client.apk]: ???

File: defense.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Defense
```
Comment (31Nathan) [inline defense.md]: Want to push this ?

**Review by 31Nathan — CHANGES_REQUESTED** [review]

File: Web/release/client.apk (hunk 1 above)
Comment (osvegn) [reply Web/release/client.apk]: a mistake

File: defense.md (hunk 2 above)
Comment (osvegn) [reply defense.md]: a mistake too

**Review by JohanCDev — APPROVED** [review]
Lgtm

**Review by JohanCDev — DISMISSED** [review]
Ntr

**Review by JohanCDev — DISMISSED** [review]
Lgtm

**Review by 31Nathan — DISMISSED** [review]

**Review by CedricCORGE — DISMISSED** [review]
lgtm

**Review by CedricCORGE — APPROVED** [review]
lgtm

**Review by JohanCDev — APPROVED** [review]
Lgtm

**Pull request merged**